frontend/src/libs/types/sidebar-route.type.ts

```diff
@@ -0,0 +1,11 @@
+import { type AppRoute } from '#libs/enums/enums.js';
+
+import { type ValueOf } from './types.js';
```

Comment:
It will cause circular dependency

Reply:
@v0ldemar01 Could you explain how can I avoid it?

Reply:
As a [solution](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/117#discussion_r1310331902)

Reply:
Thanks, I got it

Reply:
Done